tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2920,6 +2947,24 @@ def rgb_to_yiq(images):
 
   Returns:
     images: tensor with the same shape as `images`.
+
+    Usage Example:
+    
+    >>> image = [[[1.0, 2.0, 3.0],[4.0, 5.0, 6.0]],...
```

Comment:
Use integer values and a 2-D array if possible here and everywhere else.

Reply:
I have made the changes. But integer values and a 2D array isn't possible because 
of the kernel structures of those I haven't changed. The kernel looks like this
`_rgb_to_yiq_kernel = [[0.299, 0.59590059, 0.2115],
                      [0.587, -0.27455667, -0.52273617],
                      [0.114, -0.32134392, 0.31119955]]` and the others which i haven't changed are similar. So using a 2D array will raise a Matrix-size incompatible error